src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -1501,12 +1526,27 @@ internal void UninstallPackage(string packageId)
                         action,
                         CancellationToken.None);
                 },
-                nugetUi => SetOptions(nugetUi, NuGetActionType.Uninstall));
+                nugetUi => SetOptions(nugetUi, NuGetActionType.Uninstall, new[] { package }));
         }
 
+        /// <summary>
+        /// For Apex tests
+        /// </summary>
+        /// <param name="packages"></param>
         internal void UpdatePackage(List<PackageIdentity> packages)
         {
-            if (packages.Count == 0)
+            var items = _packageList.PackageItems.Join(
+                packages,
+                p => new PackageIdentity(p.Id, p.Version),
+                i => i,
+                (p, i) => p);
+
+            UpdatePackage(items.ToArray());
+        }
+
+        internal void UpdatePackage(PackageItemViewModel[] packages)
```

Comment:
Why do we need an array here? If there's no reason, we can eliminate a `ToArray()` above.

Reply:
The array was needed because command parameters where this is called is an array. This method signature has changed.